runtime.ts

```diff
@@ -1142,16 +1142,27 @@ module J2ME {
     return null;
   }
 
+  function reportError(method, key) {
+    return function() {
+      try {
+        return method.apply(this, arguments);
+      } catch (e) {
+        stderrWriter.errorLn("Native " + key + " throws:" + e);
```

Comment:
With release=0 (the default), I see a bunch of these calls, but *e* stringifies to `[object Object]` for Java exceptions, which isn't very informative. Can we stringify them to something more detailed? Something like this should work:

```
      } catch (e) {
        var msg = e;
        if (e.klass) {
          msg = e.klass.classInfo.className + " " +
                util.fromJavaString(CLASSES.getField(e.klass.classInfo, "I.detailMessage.Ljava/lang/String;").get(e));
        }
        stderrWriter.errorLn("Native " + key + " throws: " + msg);
        throw e;
      }
```